coverage_assessment.R

```diff
@@ -42,6 +42,140 @@ nut <- get_data()
 # Problem 3: Prevalence of acute undernutrition/malnutrition ----
 
 
+SEX <- ifelse(nut$sex == 1, "Male", "Female") ##Assigning male and female to sex
+
+OEDEMA <- ifelse(nut$oedema == 1, "Yes", "No") ## Assigning yes and No to oedema
+
+COUNTY <- nut$county ##Assigning nut$county to COUNTY
+
+Den <- sum(nut$age >=6 & nut$age<60, na.rm = TRUE) ## Assigning the total number of children between 6-60 to Den
+
+
+####NUMBER AND PREVALENCE OF MAM CASES
+
+
+Mod_MUAC <- with (nut, muac>= 11.5 & muac <12.5 & age >=6 & age<60) ## Assigning the moderate MUAC criteria for children between 6-60 for Mod_MUAC
+
+MAM <- table (SEX, OEDEMA, Mod_MUAC, COUNTY, useNA = "ifany") ## Assigning MAM to table 
+
+data.frame(MAM) ##CREATING A DATA FRAME FOR TOTAL MAM CASES
+
+
+MAM_num_F1 <- MAM ["Female", "No", "TRUE","Urban Montserrado"] ## TOTAL MAM FEMALES IN URBAN MONTSERRADO
+
+MAM_num_F2 <- MAM ["Female", "No", "TRUE","Grand Bassa"] ## TOTAL MAM FEMALES IN GRAND BASSA
+
+MAM_num_M1 <- MAM ["Male", "No", "TRUE","Urban Montserrado"] ## TOTAL MAM MALES IN URBAN MONTSERRADO
+
+MAM_num_M2 <- MAM ["Male", "No", "TRUE","Grand Bassa"] ## TOTAL MAM MALES IN GRAND BASSA
+
+MAM_number <- data.frame(Sex = c("Female", "Female", "Male", "Male"),County = c("Urban Montserrado", "Grand Bassa", "Urban Montserrado", "Grand Bassa"), Number_of_MAM = c(MAM_num_F1, MAM_num_F2, MAM_num_M1, MAM_num_M2)) ## CREATING DATA FRAME FOR MAM CASES BY SEX AND COUNTY
+
+MAM_number_table <- xtabs(Number_of_MAM ~ Sex + County, data = MAM_number) ##CREATING A TABLE BY SEX AND COUNTY
```

Comment:
it is interesting here that you decide to use `xtab` after taking all the effort to do all the lines of code above. You could have used `xtab` right from the start and saved yourself at least 10 lines of code to get to the same answer.

Also, you are still not disaggregating the results by baseline and endline. Your output results are still for combined baseline and endline. Please add grouping for this.